helper/reports.go

```diff
@@ -55,3 +57,26 @@ func ProcessListReports(fileID int) error {
 	return nil
 
 }
+
+func ProcessDownloadReportCSV(reportID int, outputFilePath string) error {
+	ctx := context.Background()
+	client := getClient()
+	downloadUrl, err := client.Reports.GetDownloadUrlCSV(ctx, reportID)
+	if err != nil {
+		return err
+	}
+
+	resp, err := client.Reports.DownloadReportData(ctx, downloadUrl)
+	if err != nil {
+		return err
+	}
+	if outputFilePath != "" {
+		_, err := client.Reports.WriteReportDataToFile(resp.Body, outputFilePath)
+		if err != nil {
+			return errors.New(fmt.Sprintf("Failed to download report. Error: %v", err))
+		}
+	}
+	err = client.Reports.WriteReportDataToTerminal(resp)
```

Comment:
I'm not sure if we should print date to the terminal in both cases. I think we should only do one thing. Either write to file or write to stdout.

Reply:
Yes, early exit is added with no error